### PR TITLE
fix attachtest case

### PR DESCRIPTION
**Pull request by songkaiape**
use javac -g to compile the case

### CODE REVIEW

File: src/debug/attach-test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('Attach test', () => {
         return (async () => {
             config = new AttachTest();
             DATA_ROOT = path.join(ROOT, config.workspaceRoot);
-            let cmdStr = 'cd dir&&cd src&&javac ./test/attachdebug.java&&java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug';
+            let cmdStr = 'cd dir&&cd src&&javac -g ./test/attachdebug.java&&java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug';
```
Comment (andxu) [inline src/debug/attach-test.js]: "`cd ${DATA_ROOT}/src && javac -g ./test/attachdebug.java && java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=1044 test.attachdebug`"

Comment (andxu) [reply src/debug/attach-test.js]: user string template to replace 
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/template_strings

Comment (andxu) [reply src/debug/attach-test.js]: add reasonable space for better code experience.

Comment (songkaiape) [reply src/debug/attach-test.js]: Got it.

**Review by andxu — APPROVED** [review]

**Pull request merged**